httpbakery/client.go

```diff
@@ -88,11 +95,27 @@ func NewHTTPClient() *http.Client {
 	return &c
 }
 
-// WaitResponse holds the type that should be returned
-// by an HTTP response made to a WaitURL
-// (See the ErrorInfo type).
-type WaitResponse struct {
-	Macaroon *macaroon.Macaroon
+// Client holds the context for making HTTP requests
+// that automatically acquire and discharge macaroons.
+type Client struct {
```

Comment:
It'd be  really good if the zero value just  worked, perhaps have a httpbakery.DefaultHTTPClient that it can use?